.chloggen/k8sobjectsreceiver-fix-observedtimestamp.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'bug_fix'
```

Comment:
Nit: I think technically it's an `enhancement`, since missing timestamp is not a bug in itself?

Reply:
Ops!
You're right, just changed it 😄